### PR TITLE
Added flake8 to pre-commit

### PR DESCRIPTION
**Pull request by Raalsky**

### CODE REVIEW

File: src/neptune_sacred/impl/__init__.py [hunk 1]
```diff
@@ -25,18 +25,10 @@
...
-        expect_not_an_experiment,
-        verify_type,
-    )
+    from neptune.new.integrations.utils import expect_not_an_experiment
 except ImportError:
     # neptune-client>=1.0.0 package structure
-    import neptune
-    from neptune.integrations.utils import (
-        expect_not_an_experiment,
-        verify_type,
```
Comment (AleksanderWWW) [inline src/neptune_sacred/impl/__init__.py:38]: why was that removed?

Comment (Raalsky) [reply src/neptune_sacred/impl/__init__.py:38]: Unused import

**Review by AleksanderWWW — APPROVED** [review]
